### DISCUSSION TITLE
What is the write strategy for volume

### DISCUSSION THREAD

**Discussion by peiqi92** [seaweedfs/seaweedfs#5214] [Q&A]
As a novice using seaweedFS, I have a question: When I deploy 3 volume service nodes, each node is assigned 8 volumes by default. When I upload files concurrently, which volume service nodes and volumes will these files be written to, and what is the allocation strategy for writing? I look forward to your reply.

**Comment by chrislusf** [comment]
It's random.

what do you think is the ideal strategy?

**Reply by peiqi92** [reply]
If the write operation is completely random, will some volume nodes have high resource usage (such as disk,cpu,memory, and network bandwidth) and some nodes have insufficient load?
Whether there is a load balancing policy?

**Reply by chrislusf** [reply]
this would happen only if there are a few volumes.

**Reply by peiqi92** [reply]
I can't understand how random writing actually works，and I can't find a description of the write strategy in the wiki documentation.Would you consider supporting the following write policies like fastdfs?
(1) Round robin: polling among all group nodes.
(2) Specified group.
(3) Load balance: Select the group with the largest free space to upload files.

**Reply by chrislusf** [reply]
SeaweedFS volumes can be moved around to balance the load/storage. There are no need for such policies.
